File: BeeCrowd/Python/beecrowd_1097.py
j, i, count = [7,7], 1, 0
while (not(i == 11  and j[1] == 17)):
    print("I=%d J=%d" %(i,j[1]))
    if (count == 2):
        j[0] += 2
        i += 2
        j[1], count = j[0], 0
    else:
        j[1] += -1
        count += 1

File: BeeCrowd/Cpp/beecrowd_1097.cpp
#include <stdio.h>
 
int main() {
 
    int j[] = {7,7}, i = 1, count = 0;
    while (!(i == 11  && j[1] == 17)) {
        printf("I=%d J=%d\n",i,j[1]);
        if (count == 2) {
            j[0] += 2;
            j[1] = j[0];
            i += 2;
            count = 0;
        }
        else { --j[1]; ++count; }
    }
    
    return 0;
}
